scripts/irods/database_upgrade.py

```diff
@@ -69,6 +69,12 @@ def run_update(irods_config, cursor):
         database_connect.execute_sql_statement(cursor, "create index idx_data_main7 on R_DATA_MAIN (resc_id);")
         database_connect.execute_sql_statement(cursor, "create index idx_data_main8 on R_DATA_MAIN (data_is_dirty);")
 
+        timestamp = '{0:011d}'.format(int(time.time()))
+        rows = database_connect.execute_sql_statement(cursor, "select user_id from R_USER_MAIN where user_name in ('rodsadmin', 'public') and user_type_name = 'rodsgroup';").fetchall()
```

Comment:
don't we want a query to find accounts that are *NOT* in the `R_USER_GROUP` table?  so, if they had already 'fixed' this manually, there would be nothing to do.  and if they had somehow added others... they would get found here as well?

Reply:
Will adjust it.

Reply:
the SQL has been adjusted and should handle all cases gracefully.